kubernetes/crds/crd.yaml

```diff
@@ -39,11 +39,13 @@ spec:
               properties:
                 host:
                   type: string
-                key:
+                passwordSecret:
                   type: string
                 port:
                   type: integer
-                secret:
+                secretKey:
+                  type: string
+                user:
```

Comment:
As I mentioned above for me the naiming is not straigtforward, I beleive that something like:
`passwordSecretName`
`passwordSecretKey`
or even something like:
```
passwordSecretRef:
  name:
  key:
```
will be more clear